linux/LinuxProcessList.c

```diff
@@ -1003,6 +1003,44 @@ static inline void LinuxProcessList_scanMemoryInfo(ProcessList* this) {
    fclose(file);
 }
 
+static inline void LinuxProcessList_scanZramInfo(LinuxProcessList* this){
+   unsigned long long int totalZram = 0;
+   unsigned long long int usedZramComp = 0;
+   unsigned long long int usedZramOrig = 0;
+
+   char mm_stat[32];
+   char disksize[32];
+
+   FILE* file;
+   unsigned int i = 0;
+   snprintf(mm_stat,32,"/sys/block/zram%u/mm_stat",i);
+   snprintf(disksize,32,"/sys/block/zram%u/disksize",i);
```

Comment:
These buffers can overflow for large `i`. Always assume integer formatting to produce up to 10 digits.